include/minuteman.hrl

```diff
@@ -9,6 +9,12 @@
 -author("sdhillon").
 -author("Tyler Neely").
 
+-record(ct_timer, {
```

Comment:
If so, then put it in an -ifdef(TEST). I'm surprised we didn't get a record unused error.